pkgs/tools/misc/fontforge/default.nix

```diff
@@ -9,22 +9,27 @@
 
 stdenv.mkDerivation rec {
   name = "fontforge-${version}";
-  version = "20160404";
+  version = "20170730";
 
   src = fetchFromGitHub {
     owner = "fontforge";
     repo = "fontforge";
     rev = version;
-    sha256 = "15nacq84n9gvlzp3slpmfrrbh57kfb6lbdlc46i7aqgci4qv6fg0";
+    sha256 = "15k6x97383p8l40jvcivalhwgbbcdg5vciyjz6m9r0lrlnjqkv99";
   };
 
-  patches = [(fetchpatch {
-    name = "use-system-uthash.patch";
-    url = "http://pkgs.fedoraproject.org/cgit/fontforge.git/plain/"
-      + "fontforge-20140813-use-system-uthash.patch?id=8bdf933";
-    sha256 = "0n8i62qv2ygfii535rzp09vvjx4qf9zp5qq7qirrbzm1l9gykcjy";
-  })];
-  patchFlags = "-p0";
+  patches = [
+    ./fontforge-20140813-use-system-uthash.patch
+
+    (fetchpatch {
+       url = "https://github.com/fontforge/fontforge/compare/${version}...volth:rb-${version}.patch";
```

Comment:
Any reason not to commit the patch into NixPkgs (the link would be a comment)? Mabe even under the same naming scheme, to avoid forgetting to bump it.